sup3r/models/generator_discriminator.py

```diff
@@ -0,0 +1,781 @@
+# -*- coding: utf-8 -*-
+"""Sup3r model software"""
+import copy
+import os
+import time
+import logging
+import numpy as np
+import pprint
+import pandas as pd
+import tensorflow as tf
+from tensorflow.keras import optimizers
+from warnings import warn
+
+from sup3r.models.generator import Sup3rGen
+from sup3r.utilities import VERSION_RECORD
+
+
+logger = logging.getLogger(__name__)
+
+
+class Sup3rGan(Sup3rGen):
```

Comment:
Im wondering if we should still call this file base.py, since it includes the "base" Gan model. This would avoid all the renamed imports, although that alone shouldn't prevent an improvement.

Reply:
Yes, now that I think more about it, aside from the conditional moments models, all the models are GANs, so we can probably keep it as `base`